packages/fields-wysiwyg-tinymce/views/Field/Field.js

```diff
@@ -0,0 +1,50 @@
+/* eslint-disable import/no-extraneous-dependencies */
+/** @jsx jsx */
+
+import { jsx } from '@emotion/core';
+import { Component } from 'react';
+
+import { FieldContainer, FieldLabel } from '@arch-ui/fields';
+import { ShieldIcon } from '@arch-ui/icons';
+import { colors } from '@arch-ui/theme';
+
+import Editor from '../Editor';
+
+export default class WysiwygField extends Component {
+  onChange = value => {
+    if (typeof value === 'string' && value !== this.props.value) {
+      this.props.onChange(value);
+    }
+  };
+  render() {
+    const { autoFocus, field, error, value: serverValue } = this.props;
+    const value = serverValue || '';
+    const htmlID = `ks-input-${field.path}`;
+    const canRead = !(error instanceof Error && error.name === 'AccessDeniedError');
+
+    if (!canRead) return null;
+
+    return (
+      <FieldContainer>
+        <FieldLabel
+          htmlFor={htmlID}
+          css={{
+            display: 'flex',
+            flexDirection: 'row',
+            justifyContent: 'space-between',
+          }}
+        >
+          {field.label}
+        </FieldLabel>
+        <div css={{ display: 'flex', flex: 1 }}>
+          <Editor
+            value={canRead ? value : undefined}
```

Comment:
```suggestion
            value={value}
```

Line 25 means this can never be false at this point.